src/libman/Commands/CleanCommand.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    /// <summary>
+    /// Defines the clean command to allow user to clean the files installed via libman.
+    /// </summary>
+    internal class CleanCommand : BaseCommand
+    {
+        public CleanCommand(IHostEnvironment hostEnvironment, bool throwOnUnexpectedArg = true)
+            : base(throwOnUnexpectedArg, "clean", Resources.CleanCommandDesc, hostEnvironment)
+        {
+        }
+
+        public override string Remarks => Resources.CleanCommandRemarks;
+
+        protected override async Task<int> ExecuteInternalAsync()
+        {
+            Manifest manifest = await GetManifestAsync();
+            IEnumerable<ILibraryInstallationResult> result = manifest.Clean((s) => HostEnvironment.HostInteraction.DeleteFiles(s));
```

Comment:
Task<IEnumerable<ILibraryInstallationResult>> CleanAsync

Reply:
Is this a new method being added to the Manifest?